docs/modules/mssql.md

```diff
@@ -49,6 +49,7 @@ Provisions a new SQL Server and a new database upon that server. The new dbms an
 | `firewallRules[n].name` | `string` | Specifies the name of the generated firewall rule |Y | |
 | `firewallRules[n].startIPAddress` | `string` | Specifies the start of the IP range allowed by this firewall rule | Y | |
 | `firewallRules[n].endIPAddress` | `string` | Specifies the end of the IP range allowed by this firewall rule | Y | |
+| `connectionPolicy` | `string` | Refer [here](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-connectivity-architecture#connection-policy) to specify a connection policy. Valid values are "Redirect", "Proxy", "Default" | N | |
```

Comment:
For provisioning step, a default value should be specified. The same below.

Reply:
It is an optional parameter. The provision step can be skip if the field is empty like an update step. Does it make sense?

Reply:
It's developer's knowledge, however, if I'm a new user, I'll be confused about what will happen if I don't provide this non-required value, a brief clarification will be better.

Reply:
Good point thanks. Updated now.